### DISCUSSION TITLE
Ubuntu下面运行避坑

### DISCUSSION THREAD

**Discussion by midpoint** [tw93/Pake#245] [Ideas]
在ubuntu下面按照指南运行打包，遇到很多问题，主要是库没安装完整，运行下面的命令会少很多麻烦：
sudo apt install libdbus-1-dev libgtk-3-dev libsoup2.4-dev libjavascriptcoregtk-4.0-dev libwebkit2gtk-4.0-dev

**Comment by tw93** [comment]
是指命令行pake-cli的使用过程吗？

**Comment by midpoint** [comment]
是的 ，之前没有配置好，基本的系统，运行命令：
pake 网址
出现各种错误……

**Reply by tw93** [reply]
谢谢指出，感觉你可以给这两个文档提一个PR，相当于加一个 Ubuntu 系统需要额外安装的东西
https://github.com/tw93/Pake/blob/master/bin/README.md
https://github.com/tw93/Pake/blob/master/bin/README_EN.md

**Reply by tw93** [reply]
已添加到文档
